### PR TITLE
XPersona Dataset

### PR DESCRIPTION
**Pull request by Zheng0428**
**Patch description**
Add XPersona dataset ([https://github.com/HLTCHKUST/Xpersona](url))

**Testing steps**
Display training set in default language (English)：
`parlai display_data -t xpersona`
Display train set for a specific language :
`parlai display_data -t xpersona:Fr`
`parlai display_data -t xpersona:En`
`parlai display_data -t xpersona:Zh`
`parlai display_data -t xpersona:Ko`
`parlai display_data -t xpersona:Ja`
`parlai display_data -t xpersona:It`
`parlai display_data -t xpersona:Id`
Display test set for a specific language：
`parlai display_data -t xpersona:Zh -dt test`
Display valid set for a specific language：
`parlai display_data -t xpersona:Zh -dt valid`
Display train set for a specific language：
`parlai display_data -t xpersona:Zh -dt train`

### CODE REVIEW

File: parlai/tasks/xpersona/build.py [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+                    for utterance_A, utterance_B in dialogs:
+                        line_num += 1
+                        f_write.write(
+                            str(line_num)
+                            + ' '
+                            + utterance_A
+                            + '\t'
+                            + utterance_B
+                            + '\n'
+                        )
```
Comment (stephenroller) [inline parlai/tasks/xpersona/build.py:199]:
```suggestion
                        f_write.write(f"{line_num} {utterance_A}\t{utterance_B}\n")
```

**Review by stephenroller — APPROVED** [review]
lgtm, thanks for adding this!

**Comment by stephenroller** [comment]
Waiting on tests to pass since I changed that one line. Ping if we forget to smash merge.

**Comment by zhangmozhi** [comment]
Gentle ping on this :-)
